package.json

```diff
@@ -2,7 +2,11 @@
   "name": "fixed-data-table-2",
   "version": "2.0.0-beta.7",
   "description": "A React table component designed to allow presenting thousands of rows of data.",
-  "main": "main.js",
+  "main": "dist/cjs/fixed-data-table-2.js",
+  "module": "dist/es/fixed-data-table-2.js",
```

Comment:
Notice we have 2 different entry points, one for common JS, and the other one for ES2015 modules. (https://github.com/rollup/rollup/wiki/pkg.module)